src/js/quizapiWrapper.js

```diff
@@ -0,0 +1,69 @@
+import fetch from "node-fetch";
+export class QuizapiWrapper {
+    constructor(category) {
+        this.apiKey = "EAco9tkerMBqitin1irWhCDipOxYHxJ5A5xe1nqQ"
+        this.data;
+        this.checkCategory(category) 
+        this.url = `https://quizapi.io/api/v1/questions/?apiKey=${this.apiKey}&tags=${this.category}`;
+    };
+
+    async getData() {
+        
+        const callApi = (url) => {
+            const data = fetch(url)
+            .then(res => res.json())
+            .then(json => json)
+            .catch(err => console.error('Error:', err));
+            return data;
+        }
```

Comment:
We can simplify this:
```
const callApi = (url) => (
  fetch(url)
    .then(res => res.json())
    .then(json => json)
    .catch(err => console.error('Error:', err))
);
```
Also the `.then(json => json)` does nothing at all. It returns the same data as it recieves so we can remove it safely.